corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,10 +448,9 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
                 arrow.removeClass("fa-angle-double-up");
                 arrow.addClass("fa-angle-double-down");
                 tileContent.addClass("collapsed-tile-content");
-                const scrollContainer = $(constants.SCROLLABLE_CONTENT_CONTAINER);
                 const offset = getScrollTopOffset(this.smallScreenEnabled);
-                $(scrollContainer).animate({
-                    scrollTop: scrollContainer.scrollTop() + $(e.currentTarget).parent().offset().top - offset
+                $(this.scrollContainer).animate({
```

Comment:
My understanding is that you usually reassign this to self in contexts that are run outside the current class. For example if you have a function that uses this/self and pass it to something that runs it asynchronously this might change but the self variable would refer to the original one.